draft-ietf-rats-corim.md

```diff
@@ -2219,6 +2254,32 @@ The following transformation steps are applied for both the `identity-triples` a
 
 * If the Endorsement conceptual message has a profile, the profile is copied to the `ev`.`addition`.`profile` field.
 
+#### Domain Membership Triples Transformation {#sec-dm-trans}
+
+{:dmt-enum: counter="foo" style="format Step %d."}
+
+{: dmt-enum}
+* An `dm` list entry ({{sec-dm-membership}}) is allocated.
+
+* The `cmtype` of the `members` ECT in the `dm` entry is set to `domain-member`.
+
+* The Domain Membership Triple (DMT) ({{sec-comid-triple-domain-membership}}) populates the `dm` ECTs.
+
+{:dmt2-enum: counter="dmt2" style="format %i"}
+
+{: dmt2-enum}
+* DMT.`identity`, $domain-identity-type-choice `environment-map` as under:
+
+> > **copy**(`environment-map`, `dm`.`identity`.`environment-map`)
+
+{: dmt2-enum}
+* For each e in DMT.`members`:
+
+> > **copy**(e.`environment-map`, `dm`.`members`.`environment-map`)
+
+
+* If the Endorsement conceptual message has a profile, the profile identifier is copied to the `dm`.`members`.`profile` field.
+
```

Comment:
Should there be a section in "ACS augmentation using CoMID triples" that describes how to process the dm structure so that it results in changes or additions to the ACS entries? For example, for all the members in dm.members, the ACS is searched to find environment-maps that match and the ACS.EMT.domain field is updated to include dm.members.environment-map. If the search returns NULL then what? Should a new EMT be added to the ACS? 

What happens if there are multiple matches? Do all of them get their EMT.domain fields updated? 

What happens if there is a partial match, for example only class-map, instance-id, or group-id (or combinations of the three) match?

Reply:
If the match fails, for a DMT, it is not added to ACS.

If none of the DMT succeeds in a `complete match` and a Verifier Policy requires DMT then it SHOULD fail Verification!

For partial match also, the result is a Failure!

I was planning to add this, description, once we (CoRIM team) is aligned in general to this concept!

Reply:
I am assuming that a `profile` will populate `DMT: Environment` structure identical to Evidence ECT's Environment structure.

Reply:
Added a separate section, hence resolving the comment now!